docs/src/overview/kubernetes.md

```diff
@@ -44,3 +44,112 @@ Due to the reason we use ignition in our operating system images for userdata, w
 ### machine-controller-manager-provider-metal
 
 Worker nodes are managed through Gardener's [machine-controller-manager](https://github.com/gardener/machine-controller-manager) (MCM). The MCM allows out-of-tree provider implementation via sidecar, which is what we implemented in the [machine-controller-manager-provider-metal](https://github.com/metal-stack/machine-controller-manager-provider-metal) repository.
+
+
+## Gardener with metal-stack
+
+### Introduction into Gardener
+
+[Gardener](https://gardener.cloud/) is an open-source project and a system to manage Kubernetes clusters. Based on one cluster, other K8s clusters can be created by Gardener on many different cloud providers.
+
+### Gardener glossary
+
+At first, the most important Gardener terms are explained. More information can also be found in the [glossary](https://github.com/gardener/documentation/blob/master/website/documentation/glossary/_index.md).
+
+**Garden Cluster**
+
+A dedicated Kubernetes cluster that the Gardener control plane runs in. The Kubernetes cluster can be setup e.g. with Kubespray.
+The Garden cluster can also be used as seed cluster by deploying the Gardenlet into it.
+
+**Virtual Garden**
+
+Is a virtual cluster inside the Garden cluster. The virtual cluster is node less. Is is only a control plane node / cluster with the following components:
+
+- garden kube-apiserver
+- etcd
+- kube-controller-manager
+
+More details about the value of a virtual garden can be found in the description of [garden-setup](https://github.com/gardener/garden-setup/?tab=readme-ov-file#concept-the-virtual-cluster).
```

Comment:
Probably it's not so good to reference garden-setup here because it's outdated and not maintained. They use the Gardener operator now to spin up the virtual garden. So maybe try to reference the operator: https://github.com/gardener/gardener/blob/master/docs/concepts/operator.md